packages/app/src/Util.ts

```diff
@@ -491,6 +491,20 @@ export interface NostrLink {
   encode(): string;
 }
 
+export function validateNostrLink(link: string): boolean {
+  try {
+    const parsedLink = parseNostrLink(link);
+
+    if (!parsedLink) {
+      return false;
+    }
+
+    return parsedLink.id.length === 64;
```

Comment:
Any valid ID should be 64 chars, right?

Reply:
No, the ID type for naddr, is the d-tag, maybe should tadd some tests to this?